dbt/include/global_project/macros/schema_tests/unique.sql

```diff
@@ -1,15 +1,17 @@
 
-{% macro test_unique(model, arg) %}
+{% macro test_unique(model) %}
+
+{% set column_name = kwargs.get('column_name', kwargs.get('arg')) %}
```

Comment:
In the v1 schema spec, you could declare a test on an expression, eg:

```yml
model_name:
  constraints:
    unique:
      - concat(id, '-', date)
```

The v2 spec doesn't support this sort of test. Is there a decent way to replicate this type of test in v2, or is it something that should be handled separately, eg with:
 - a custom data test
 - a custom schema test
 - just creating the column from the expression in the model, then testing that column